Source/OCMockTests/OCMockObjectMacroTests.m

```diff
@@ -363,6 +363,37 @@ - (void)testShouldThrowExceptionWhenNotUsingMockInReject
             NSException, NSInternalInconsistencyException, @"should throw NSInternalInconsistencyException exception");
 }
 
+- (void)testShouldThrowExceptionWhenAttemptingToMockNSObjectMethodInVerify
+{
+  id mock = OCMClassMock([NSString class]);
+
+  XCTAssertThrowsSpecificNamed(OCMVerify([mock description]),
+          NSException, NSInternalInconsistencyException, @"should throw NSInternalInconsistencyException exception");
+}
+
+- (void)testShouldThrowExceptionWhenAttemptingToMockNSObjectMethodInStub
```

Comment:
Does the exception invalidate the mock in some way? I don't see why these are separate tests vs. a single test with multiple checks?

Reply:
Done (and combined tests above to be the same).